### DISCUSSION TITLE
Compose splitAtom with AtomWithQuery

### DISCUSSION THREAD

**Discussion by spacepolice10** [pmndrs/jotai#2308] [Q&A] [answered]
Hi there, thank for all the hard work Jotai's community does. Is it possible to access ‘traditional’ atom inside `atomWithQuery` so it is possible to combine it with `splitAtom` (e.g. when query itself returns a list of items) ? I've some discussions related to the question, but I believe API is recently changed quite drastically

**Comment by dai-shi** [comment, marked as answer]
You should still be able to do something like this.
```js
const queryAtom = atomWithQuery(...)
const dataAtom = atom(async (get) => (await get(queryAtom)).data)
```
For splitAtom, you probably want `unwrap` too.
